add finance prolicy about proposals being public (#153)
astropy/astropy-project: finance/finance-ctte-policy.rst

   __ https://docs.google.com/document/d/193ERJ6wMNPaNjGSWbS94KHGnojpZ_zwXxWTi9L2etSo/edit#bookmark=id.bwbxtdjn2

> #. Funding proposals that are written on behalf of the Astropy Project should be public and open for comment to the Astropy community unless there is a specific reason they cannot be.  Similarly, progress reports on such grants required by the provider of the grant should also be shared publicly with the Astropy community if at all possible.

Reviewer: One sentence per line, for consistency with the rest of the document?

> #. Funding proposals that are written on behalf of the Astropy Project should be public and open for comment to the Astropy community unless there is a specific reason they cannot be.  Similarly, progress reports on such grants required by the provider of the grant should also be shared publicly with the Astropy community if at all possible.

Reviewer: ```suggestion
Funding proposals that are written on behalf of the Astropy Project should be public, and comments should be solicited from the community before the proposal is submitted, unless there are specific reasons why this is not possible.
```

>    (Agreed `2020-10-2`__.)

Reviewer 2: ```suggestion
   (Agreed `2020-10-02`__.)
```
